src/super_gradients/training/utils/output_adapters/detection_adapter.py

```diff
@@ -122,6 +165,8 @@ class DetectionOutputAdapter(nn.Module):
     >>>
     >>> yolox = nn.Sequential(yolox, output_adapter)
     >>>
+
+    Note: DetectionOutputAdapter has a common logic but different implementation to DetectionFormatAdapter in order to be serializable with TorchScript.
```

Comment:
It took me quite a bit to understand the difference in use of those both.
Lets separate them into different pages and possibly rename one of them to avoid confusion...

Reply:
I split it to 2 different modules:
format_converter.py (with class: ConcatenatedTensorFormatConverter)
output_format_adapter.py (with class: DetectionOutputFormatAdapter)